packages/commons/src/fhir/transformers/index.ts

```diff
@@ -2698,12 +2699,15 @@ const builders: IFieldBuilders = {
             if (!hasAffidavit) {
               relatedPersonResource.extension.push({
                 url: `${OPENCRVS_SPECIFICATION_URL}extension/relatedperson-affidavittype`,
+                //@ts-ignore
```

Comment:
I will be removing the certificate related fhir-builders in the "issue-record" PR